docs/docs/components.mdx

```diff
@@ -1542,7 +1542,7 @@ However, additional parameters exist that can be adapted.
 |                                 |                  | or 'inner'.                                                  |
 +---------------------------------+------------------+--------------------------------------------------------------+
 | loss_type                       | "cross_entropy"  | The type of the loss function, either 'cross_entropy'        |
-|                                 |                  | or 'margin'.                                                 |
+|                                 |                  | or 'triplet'.                                                |
```

Comment:
Since margin loss is still in the codebase, we should have it in the documentation too (can mention deprecated for that option). This topic came up when we were deprecating components like `HFTransformersNLP` and the recommendation was to only remove it once it is removed from the codebase.
Same for other parameters below.